### ISSUE TITLE
n_training_epochs multiplied by 10

### ISSUE THREAD

**Issue by davidpng** [lmcinnes/umap#675]
When I run ParametricUMAP, the number of epochs run is 10x whatever I feed into n_training_epochs.  This doesn't seem like the correct behavior.

**Comment by timsainb** [comment]
Hi David,

By default, a single epoch is broken into tenths, so it's really just 1 epoch running for every 10 'epochs' in keras.

**Comment by timsainb** [comment]
This is just so you can visualize the training loss at the sub-epoch resolution.
